internals/generators/organization/config.yml.hbs

```diff
@@ -281,6 +281,23 @@ client:
           WiFi Login Successful!
           You can now use the internet.
           You may leave this page open in case you want to log out.
+      user_info:
+        status:
+          text:
+            en: "Status"
+          value:
+            en: "Logged in"
+        email:
+          text:
+            en: "Email"
+        username:
+          text:
+            en: "Username"
+      {{#if mobile_phone_verification}}
+        phone_number:
+          text:
+            en: "Phone number"
+      {{/if}}
```

Comment:
let's not use the if here, the reason is that if mobile phone verification is enabled at a later step, the user will have to modify manually the config file which is time consuming and error prone.